packages/finance/src/liability/dto/mod.rs

```diff
@@ -0,0 +1,274 @@
+use serde::{Deserialize, Serialize};
+
+use sdk::schemars::{self, JsonSchema};
+
+use crate::{
+    coin::CoinDTO,
+    duration::Duration,
+    error::{Error, Result},
+    percent::Percent,
+};
+use currency::{lpn::Lpns, Currency};
+
+use super::Liability;
+
+mod unchecked;
+
+pub type LpnCoin = CoinDTO<Lpns>;
+
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+#[serde(try_from = "unchecked::LiabilityDTO")]
+pub struct LiabilityDTO {
```

Comment:
Please make sure all attributes match the one of the previous `struct Liability`